src/super_gradients/training/datasets/classification_datasets/cifar.py

```diff
@@ -0,0 +1,46 @@
+from typing import Optional, Callable
+
+from torchvision.transforms import Compose
+
+from common.factories.list_factory import ListFactory
+from common.factories.transforms_factory import TransformsFactory
+from super_gradients.common.decorators.factory_decorator import resolve_param
+from torchvision.datasets import CIFAR10, CIFAR100
+
+
+class Cifar10(CIFAR10):
```

Comment:
Lets add some docs, we need to at least clarify the transforms argument since its not in the parent class.